source/dashboards/views.markdown

```diff
@@ -221,3 +221,17 @@ View configuration:
     - entity: switch.decorative_lights
       image: /local/lights.png
 ```
+
+Subview configuration:
+
+```yaml
+title: Energieprijzen
+path: energieprijzen
+subview: true
+back_path: /ui-data/climate
+
+cards:
+  - type: entities
+    entities:
+      - etc
```

Comment:
etc?

Reply:
yeah, well, I guess that would be obvious. you want me to take out that line 236, or, prefer an entity
 like `sun.sun`

Reply:
Add some example, that would help :)

Reply:
done